src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md

```diff
@@ -0,0 +1,265 @@
+
+      execution: cloud
+      script: test.js
+      output: https://app.k6.io/runs/TEST_RUN_ID
+    ```
+
+    </div>
+
+7. Navigate to the URL to check your test results.  When the test is running, the test result page is shown. 
+
+   ![Insights results](/images/cloud-insights-results.png 'k6 Cloud Test Results')
+
+   Learn more about the different test result sections on the [k6 Cloud Results docs](/cloud/analyzing-results/overview).
+
+## Cloud execution options
+
+All the [k6 Options](/using-k6/options), like `--vus` and `--duration` are the same between the `k6 run` and `k6 cloud` commands. k6 aims to run the same script on different execution modes without making any script modifications.
+
+Optionally, you can define some cloud options in your k6 script.
+
+<div class="code-group" data-props='{"labels": []}'>
+
+```javascript
+export let options = {
+  ext: {
+    loadimpact: {
+      name: 'Hello k6 cloud!',
+      projectID: 123456,
+      distribution: {
+        scenarioLabel1: { loadZone: 'amazon:us:ashburn', percent: 50 },
+        scenarioLabel2: { loadZone: 'amazon:ie:dublin', percent: 50 },
+      },
+    },
+  },
+};
+```
+
+</div>
+
+| Name                  | Default                                                                                          | Description                                                                                                                                                                                                                         |
+| --------------------- | ------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| name (string)         | Optional. The name of the main script file, so something like "script.js".                       | The name of the test in the k6 Cloud UI. Test runs with the same name will be grouped together.                                                                                                                                     |
+| projectID (number)         | Optional. It is empty by default.                       | The ID of the project in which the test is assigned in the k6 Cloud UI. By default, the default project of the user default organization.                                                                                                                                       |
+| distribution (object) | Optional. The equivalent of `someDefaultLabel: { loadZone: "amazon:us:ashburn", percent: 100 }`. | How the traffic should be distributed. The keys are string labels that will be injected as [environment variables](/using-k6/environment-variables) on the appropriate nodes (matching the `loadZone`): `__ENV["someDefaultLabel"]` |
+
+### List of supported load zones
```

Comment:
We're missing "São Paulo" (`amazon:br:sao paulo`) from this list.

Reply:
https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720